### DISCUSSION TITLE
MD5 hashes for installer packages

### DISCUSSION THREAD

**Discussion by nnebeel** [rustdesk/rustdesk#9158] [General]
Can you post MD5 checksums on the release pages (e.g. https://github.com/rustdesk/rustdesk/releases/tag/1.2.7) so that we can ensure that we're getting a legitimate copy?

**Comment by Coool** [comment]
Were these hashes for the Play Store app release?

Checksums:
- **MD5**: dad9ebfb3fe8b3d9ee7c5704ae2f15f2
- **SHA-1**: d80f4beb9448220eb1e11fab785502a1803db005
- **SHA-256**: e92d44ba9b65003c7340683b0a9234f617c1b23a1dd90e89f7d04170b72bfc86
- **SHA-384**: 20bbaa31bc3c2a8c81802e4298cdaeae0ca2a5d3e06fb50356f516c593017dc2e9dd4d302a992a2907dae0afe44661fc
- **SHA-512**: 064a6651126a63ba50bcc83c0d5e0f63976819bc1884b365756e64bf9c2a5734f5a5795cca651a244436e7284a4de500729885f422543efb2f4585ad30f4a605

Are these hashes for the F-Droid app release?

Checksums:
- **MD5**: 89f61a8b4f8289aadfae656dd1d28a96
- **SHA-1**: d47ad0dcdb04e25162b77f35e206d61a39b4ffb0
- **SHA-256**: 85366b10ecadff33e10c53250444a125203908285363c27fc62a561f9660fa48
- **SHA-384**: 7bb44e61fa4184c76bc73d6b69099cc8a89d475eb74ad2bcbf60d2c1bf770426d8677f09212effad9f3d0dfd8634d173
- **SHA-512**: aac21344034e704049e31346a939ee685fae10a12f998b7ae0f864bec51ed0aaf6afc933d09b1c6c9dceca23983defa6fa824923b6ca472f801976a9881ababe

Somehow I'm stuck on version 1.3.8 with a _e92d44ba9b65003c7340683b0a9234f617c1b23a1dd90e89f7d04170b72bfc86_ **SHA-256** hash. And can't upgrade without reinstalling app using F-Droid. How I got here! 😂

Related:
- #7892
